### ISSUE TITLE
Change Ubertooth BT Address

### ISSUE THREAD

**Issue by eli80** [greatscottgadgets/ubertooth#277]
Hello!

I'm working with Ubertooh Release 2017-03-R2.

I'd like to change device's MAC address, but I can't find the way.

Programms like spooftooph and btaddr seams not working because of a 'manufacturer unknown' error.

Is there any way to achieve the goal?

Thank You

**Comment by dominicgs** [comment]
Ubertooth is not a Bluetooth device, it doesn't have a MAC address.  `btaddr` will work with many common USB Bluetooth dongles.

**Comment by eli80** [comment]
My apologies, I'm using also the HW device ubertooth 1. 
After your answet I guess i should ask the question to the HW producer (if I find a contact)
Thank you anyway

**Comment by dominicgs** [comment]
The Ubertooth One device is not a Bluetooth dongle.

**Comment by eli80** [comment]
Sorry for my incompetence, I have no experience with this kind of things.
I know Ubertooth One is a developer platform, but actually it has a MAC address, I can see it in Kali.
Since it's not a Bluetooth dongle btaddr is not working and I can understand why, but I can't understand if there's another way to change it.
I though it's address was defined in the firmware, so that's why I posted my question here.

**Comment by dominicgs** [comment]
It's not incompetence, there's definitely a steep learning curve with these tools.

Ubertooth does not have a MAC address.  When you say that you can see it in Kali, what tool are you using?

**Comment by eli80** [comment]
I'm starting to understand something.. 
The MAC address I saw was the one associated to the computer BT module.. actually I though I was using Ubertooth from Kali, but in reality I was using the native computer BT module.
I'm using Kali on a Virtual Machine and this was probably  causing some interferences  between the two modules.
What I got is that current Ubertooth, without additional customer SW, can be mainly used for sniffing BLE traffic. 
Is it correct?
Instead, its capability for Classic BT is not clear. It can't follow the frequency  hopping in this case, right?
Am I missing something?
